gk/main.c

```diff
@@ -386,33 +434,77 @@ setup_gk_instance(unsigned int lcore_id, struct gk_config *gk_conf)
 	return ret;
 }
 
+static struct flow_entry *
+add_new_flow_from_policy(
+	struct ggu_policy *policy, struct gk_instance *instance,
+	struct gk_lpm *ltbl, uint32_t rss_hash_val)
+{
+	int ret;
+	struct gk_fib *fib;
+	struct flow_entry *fe = NULL;
+
+	fib = look_up_fib(ltbl, &policy->flow);
+	if (fib == NULL || fib->action != GK_FWD_GRANTOR) {
+		/*
+		 * Drop this solicitation to add
+		 * a policy decision.
+		 */
+		RTE_LOG(ERR, GATEKEEPER,
```

Comment:
This error should be rare, and an informative message should help a lot to diagnose what happened. For example, which flow triggered it?